content/blog/2023/03/03/2023-03-08-jenkins-awards-2023-candidates.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Jenkins Contributor Awards - Nominations Open"
+tags:
+- jenkins
+- community
+- contribute
+- awards
+- cdcon
+authors:
+- kmartens27
+opengraph:
+image: image:/images/post-images/2023/02/23/2023-02-23-cdf-awards/image1.png
+links:
+discourse: true
+---
+
+image:/images/post-images/2023/02/23/2023-02-23-cdf-awards/image1.png[CDF Community Awards 2023,width=839,height=129]
+
+The 2023 Jenkins award nomination period has ended and link:https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform[voting is now open]!
+Voting will take place until March 28, when the voting period closes, and the winners will be announced at this year's link:https://events.linuxfoundation.org/cdcon-gitopscon/[cdCon].
+
+== 2023 Award categories and nominees
+
+This year we have three awards, each with several nominees:
+
+* link:https://github.com/jenkins-infra/jenkins.io/issues/6035[*Most Valuable Jenkins Advocate*]:
+** link:https://github.com/NotMyFault[Alexander Brandes]
+** link:https://github.com/slide[Alex Earl]
+** link:https://github.com/gounthar[Bruno Verachten]
+** link:https://github.com/jmMeessen[Jean-Marc Meessen]
+** link:https://github.com/krisstern[Kris Stern]
+** link:https://github.com/MarkEWaite[Mark Waite]
+* link:https://github.com/jenkins-infra/jenkins.io/issues/6033[*Most Valuable Jenkins Contributor*]:
+** link:https://github.com/NotMyFault[Alexander Brandes]
+** link:https://github.com/dduportal[Damien Duportal]
+** link:https://github.com/janfaracik[Jan Faracik]
+* link:https://github.com/jenkins-infra/jenkins.io/issues/6034[*Jenkins Security MVP*]:
+** link:https://github.com/dduportal[Damien Duportal]
+** link:https://github.com/daniel-beck[Daniel Beck]
+** link:https://github.com/dwnusbaum[Devin Nusbaum]
+** link:https://github.com/Kevin-CB[Kevin Guerroudj]
+** link:https://github.com/ValdesChe[Valdes Che Zogou]
+
+== Voting
+
+To vote for the Jenkins awards, please use link:https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform[this Google form].
+Please be aware that this form is _only_ for Jenkins awards, as other projects are hosting their own voting process.
+
+== Awards timeline 2023
+
+* Nominations close: Friday, March 3
+* Voting opens: Wednesday, March 8
+* Voting closes: Tuesday, March 28
+* Winners announced at https://events.linuxfoundation.org/cdcon-gitopscon/[cdCon + GitOpsCon]: May 8 - 9, 2023
+
+== 2022 Winners
+
+View the award guidelines, definitions, and previous winners link:https://github.com/cdfoundation/foundation/blob/main/CDF%20Awards%20Guidelines.md[here].
```

Comment:
```suggestion
View the link:https://github.com/cdfoundation/foundation/blob/main/CDF%20Awards%20Guidelines.md[award guidelines, definitions, and previous winners].
```

Reply:
never make 'here' a hyperlink. it is not accessible to those using screen readers.

Reply:
Thanks @LizGaudet, I don't think that one even registered when I was reading things over, so that's fantastic to catch. I've updated it a bit so that it reads:
`You can view the award guidelines, definitions, and previous winners on the link:link:link:https://github.com/cdfoundation/foundation/blob/main/CDF%20Awards%20Guidelines.md[CD Foundation awards] page.`